src/common/libjade_shims/libjade_randombytes.c

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
Should this be MIT license?

Reply:
Thank you for catching this. I copied a portion of this this from libjade and I've changed the license to Apache-2.0 OR CC0-1.0. Although since the code is relatively straightforward I can change this MIT if that if preferable.